resources/views/components/nav-layer.blade.php

```diff
@@ -0,0 +1,57 @@
+@props(['id', 'children', 'title' => __('Menu'), 'hasParent' => false, 'tag' => 'form', 'parentUrl' => ''])
+@slots(['headerbutton'])
+
+<x-rapidez::slideover.mobile
+    :title="(string) $title"
+    :$id
+    :$hasParent
+    :$tag
+>
+    <x-slot:headerbutton>
+        <div class="absolute left-0 top-1/2 -translate-y-1/2 cursor-pointer text-white">
+            @include('rapidez-statamic::navigation.header-button')
+        </div>
+    </x-slot:headerbutton>
+    <div class="flex w-full flex-1 flex-col bg-inactive-100">
+        <ul class="mt-5 flex flex-col divide-y border-y bg-white">
+            @if ($hasParent && $parentUrl)
+                <li>
+                    <a href="{{ $parentUrl }}" class="normal flex items-center justify-between p-5 font-semibold">
+                        @lang('Go to :item', ['item' => strtolower($title)])
+                    </a>
+                </li>
+            @endif
+            @foreach ($children ?: [] as $child)
+                @php
+                    $url = $child['url'];
+                    if (isset($child['linked_category'])) {
+                        $url = Str::start($child['linked_category']->value()['url_path'] ?? '', '/');
+                    }
+                @endphp
+                <li class="relative">
+                    @if ($child['title'] ?? '')
+                        <a href="{{ $url }}" class="flex items-center justify-between p-5 font-semibold">
+                            {{ $child['title'] }}
+                            @if ($child['children'])
+                                <x-heroicon-o-chevron-right class="size-4" />
+                            @endif
+                        </a>
+                    @endif
+                    @if ($child['children'])
+                        @php($childId = uniqid(Str::snake("{$child['title']}" ?? '')))
```

Comment:
this is using uniqid but it adds snake case to the title for whenever there are spaces in the title
![image](https://github.com/user-attachments/assets/8db2221a-6249-45f4-b043-0c90b973f61c)

I want to use the title here since it makes it more clear what id's and for's are linked to each other in your browser.
Could of course remove that part and only have uniqid but that would make the id a bit abstract and thus hard to debug if needed